SECURITY.md

```diff
@@ -29,3 +35,89 @@ If you are editing this document please feel encouraged to change this format to
 - #993 Add restricted permissions to RocksDB - 1.4.5 - Anonymous security review
 - #740 Add hardening compile flags and `-fPIE` - 1.4.1 - Anonymous security review
 - #300 Add restricted permissions to osqueryd logging - 1.0.4
+
+## Security Requirements Met by Functionality
+
+### Confidentiality
+
+#### No Retrieval of Sensitive User-owned Data
```

Comment:
This is false. Let's not include this header/case right now. We can have a longer blueprint discussion about this issue in tandem.

Reply:
Ok I know it would be false if it were a claim. But I was thinking this document would document intents and considerations, without making guarantees — like, if your requirement is to not spy on users, osquery gives you these settings (denylisted tables, off-by-default sources, etc). With a healthy amount of caveats, because you're right that we can't claim it to be impossible.

Thoughts? Or do you still want to just axe it entirely.

Reply:
I'd axe it. There are way too many caveats to try to make this meaningful.

This is something that a TLS server vendor needs to think long and hard about. (<cough>)

Reply:
Actually... Hrm. Pairing with some other comments, I think there's a place to talk about our security goals, and we try to avoid arbitrary file read.

Reply:
Ok, I'll punt this to a blueprint issue or office hours discussion. Removed for now.